lib/utils/addSemicolonForEditInfo.mjs

```diff
@@ -0,0 +1,34 @@
+/** @import { Node as PostcssNode } from 'postcss' */
+
+/**
+ * Adjust the text in EditInfo to include a semicolon when needed.
+ *
+ * @param {PostcssNode} node
+ * @param {{range: [number, number], text: string}} fixData
+ * @returns {{range: [number, number], text: string}}
+ */
+export default function addSemicolonForEditInfo(node, fixData) {
+	if (!node.source) return fixData;
+
```

Comment:
```suggestion
```

Code coverage check revealed that this test isn't needed. I had overlooked this before.